src/uct/ib/base/ib_device.c

```diff
@@ -386,6 +387,22 @@ ucs_status_t uct_ib_device_port_check(uct_ib_device_t *dev, uint8_t port_num,
         return UCS_ERR_UNSUPPORTED;
     }
 
+    if (md->check_subnet_filter && uct_ib_device_is_port_ib(dev, port_num)) {
+        union ibv_gid gid;
+        ucs_status_t ret;
```

Comment:
code style:
1. pls define the vars in the beginning of the function (not block)
2. pls use "ucs_status_t status" rather than "ucs_status_t ret"